src/clr-angular/accordion/providers/accordion.service.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+import { AccordionModel, AccordionPanelModel } from '../models/accordion.model';
+import { AccordionStrategy } from '../enums/accordion-strategy.enum';
+
+@Injectable()
+export class AccordionService {
+  protected accordion = new AccordionModel();
+  protected readonly _panelsChanges = new BehaviorSubject<AccordionPanelModel[]>(this.accordion.panels);
+
+  getPanelChanges(panelId: string): Observable<AccordionPanelModel> {
+    return this._panelsChanges.pipe(map(panels => panels.find(s => s.id === panelId)));
```

Comment:
Shouldn't it `filter` instead of `map`?

Reply:
No, we don't wan't to filter out the observable events, any time the list is updated we map to return just the latest version of the single item we care about. It's kind of like a ngrx select statement just without the optimizations.